### DISCUSSION TITLE
xsbug SERIAL feature

### DISCUSSION THREAD

**Discussion by costa-victor** [Moddable-OpenSource/moddable#1056] [Q&A] [answered]
Hello there,

My question is related to this SERIAL feature at the xsbug **Preferences:**
![image](https://user-images.githubusercontent.com/35344574/222494154-85f8c93b-db8b-4417-a7bf-ad62e9d3f7a1.png)

I have tried to find a documentation for this, but I wasn't able to find it. Is there any documentation or video demonstrating this feature, what it does, configuration and so on?

**Comment by phoddie** [comment, marked as answer]
Hello. No, this is an experimental feature which is not currently documented or supported.

**Reply by costa-victor** [reply]
Understood, thanks @phoddie
